erpnext/accounts/doctype/accounts_settings/accounts_settings.json

```diff
@@ -422,14 +425,29 @@
    "fieldname": "round_row_wise_tax",
    "fieldtype": "Check",
    "label": "Round Tax Amount Row-wise"
+  },
+  {
+   "fieldname": "section_break_rwpq",
+   "fieldtype": "Section Break",
+   "label": "Exchange Gain Or Loss Journal"
+  },
+  {
+   "default": "0",
+   "fieldname": "post_gain_loss_journal_on_reconciliation_date",
```

Comment:
Since this is only in the reco tool we can give a popup and default it to the current date, and users can choose to update the date as per their requirement, a setting for this seems rather unnecessary

Reply:
I can see remove the configuration part. 
But, I still think the default should be the payment entry date. Since, the date field in the popup is editable they should be able to change it as needed.